enterprise_access/apps/api/v1/views.py

```diff
@@ -68,6 +73,25 @@ def decoded_jwt(self):
     def lms_user_id(self):
         return self.decoded_jwt.get('user_id')
 
+    def _validate_subsidy_request_uuids(self, subsidy_request_uuids):
+        """
+        Raises a SubsidyRequestError if subsidy UUIDs are not preset or invalid.
+
+        subsidy_request_uuids should be a list of valid uuid strings
+        """
+        if not subsidy_request_uuids:
+            error_msg = 'You must provide subsidy request UUIDs to be approved.'
+            logger.exception(error_msg)
+            raise SubsidyRequestError(error_msg, status.HTTP_422_UNPROCESSABLE_ENTITY)
```

Comment:
Should these be HTTP_400_BAD_REQUESTs? Alternatively if you wanted to you could create a serializer and put the validation logic in validate(), which should automatically raise 400s.

Reply:
yeah i can change to HTTP_400_BAD_REQUESTs